app/pages/settings/email.jsx

```diff
@@ -251,6 +259,40 @@ class EmailSettingsPage extends React.Component {
               onChange={handleInputChange.bind(this.props.user)}
             />
           </AutoSave>
+          <div>
+            {(isEmailValid)
+              ? <div>
+                  <i className='fa fa-check-circle' style={{ color: '#51db72' }} />
+                  {' '}
+                  <Translate content="emailSettings.general.emailValid" />
```

Comment:
This text isn’t linked to the form input, so will be missed by a screen reader in forms mode. You can link multiple labels to form inputs with `aria-labelledby` or `aria-describedby`. 

The GDS example for [input hint text](https://design-system.service.gov.uk/components/text-input/#hint-text) shows how to mark up an accessible text input with extra labels.